src/_settings.scss

```diff
@@ -0,0 +1,763 @@
+/* ============================================================================
+   SETTINGS
+   ========================================================================= */
+
+
+/**
+ * All of Scally's settings (Sass variables).
+ *
+ * The settings are broken down into two categories:
+ *
+ * - **Global**.....settings for the entire Scally library.
+ * - **Local**......settings for a particular Scally partial.
+ *
+ * All settings start with a `scally` namespace followed by one hyphen:
+ *
+   $scally-
+ *
+ * This namespace prevents your project settings and Scally's settings from
+ * clashing and allows you to easily see what is coming from Scally and what
+ * isn't.
+ *
+ * In addition to the `scally` namespace we also have namespaces to identify
+ * **Global** and **Local** settings:
+ *
+ * **Global** settings include a letter `g` to easily identify them:
+ *
+   $scally-g-
+ *
+ * **Local** settings include the name of the relevant partial they belong to.
+ * For example, settings for the `_base.scss` and `_grid.scss` partials look
+ * like this:
+ *
+   $scally-base-
+   $scally-grid-
+ *
+ * ----------------------------------------------------------------------------
+ *
+ * CONTENTS:
+ *
+ * 1.  Colours (Global)
+ * 2.  Typography (Global)
+ * 3.  Spacing (Global)
+ * 4.  Breakpoints (Global)
+ * 5.  Z-Index system (Global)
+ * 6.  HiDPI units (Global)
+ * 7.  HTML textual `input`s (Global)
+ * 8.  Base (Local)
+ * 9.  Container (Local)
+ * 10. Grid (Local)
+ * 11. Helpers (Local)
+ */
+
+
+
+
+/* 1. Colours (Global)
+   ========================================================================= */
+
+/**
+ * Black and white.
+ */
+
+$scally-g-color-white: #fff !default;
+
+$scally-g-color-black: #000 !default;
+
+
+/**
+ * Text.
+ */
+
+// Base (global text colour)
+$scally-g-color-text-base: #444 !default;
+
+// Heading
+$scally-g-color-text-heading: $scally-g-color-text-base !default;
+
+
+/**
+ * Links.
+ */
+
+$scally-g-color-link: #0000ff !default;
+
+$scally-g-color-link-hover: #ff0000 !default;
+
+
+
+
+/* 2. Typography (Global)
+   ========================================================================= */
+
+/**
+ * Base sizes.
+ */
+
+$scally-g-font-size: 16 !default;
+
+// Decrease
+$scally-g-font-size-small: $scally-g-font-size - 1 !default;
+
+$scally-g-font-size-x-small: $scally-g-font-size - 2 !default;
+
+$scally-g-font-size-xx-small: $scally-g-font-size - 3 !default;
+
+$scally-g-font-size-xxx-small: $scally-g-font-size - 4 !default;
+
+// Increase
+$scally-g-font-size-large: $scally-g-font-size + 2 !default;
+
+$scally-g-font-size-x-large: $scally-g-font-size + 3 !default;
+
+$scally-g-font-size-xx-large: $scally-g-font-size + 4 !default;
+
+$scally-g-font-size-xxx-large: $scally-g-font-size + 5 !default;
+
+
+/**
+ * Heading sizes.
+ */
+
+$scally-g-font-size-heading: $scally-g-font-size * 1.5 !default;
```

Comment:
Yeah, I was trying to decide if the math worked out for varying sizes, or if it was worth the cognitive load of having to work out the math?